plugins/modules/sns_topic.py

```diff
@@ -24,6 +24,12 @@
       - The name or ARN of the SNS topic to manage.
     required: true
     type: str
+  topic_type:
+    description:
+      - The type of topic that should be created. Either Standard for FIFO (first-in, first-out)
+    choices: ['standard', 'fifo']
+    default: 'standard'
+    type: str
```

Comment:
Could you please specify `version_added: 2.0.0`?

Reply:
Done